database/src/test/java/com/pip/chatbot/repository/DslContextFactory.java

```diff
@@ -0,0 +1,17 @@
+package com.pip.chatbot.repository;
+
+
+import org.jooq.DSLContext;
+import org.jooq.impl.DSL;
+
+import java.io.IOException;
+
+public class DslContextFactory {
+
+
+    public DSLContext getDslContext() throws ClassNotFoundException, IOException {
+        Class.forName("org.postgresql.Driver");
```

Comment:
TO nie potrzebne ogólnie, bo przecież pod spodem ogarnie sobie driver jooq. Pomijając czytanie z propertasów to dałbym tą metodę jako statyczną, bo niepotrzebujemy tworzyć tego factory. Nie mam przecież żadnych pól